bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -156,29 +157,14 @@ class Neuron(CMakePackage):
     depends_on("python", type=("build", "run"))
     depends_on("boost", when="@8.99:+tests+coreneuron")
     depends_on("cuda", when="@8.99:+gpu")
-    depends_on("flex@2.6:", type="build", when="+nmodl")
-    depends_on("nmodl@0.4.0:", when="@8.99:+nmodl")
+    depends_on("nmodl@0.4.0:", when="@9:+nmodl")
```

Comment:
We need to depend on `nmodl` even in the `develop` version that doesn't have an `+nmodl` variant.